cloudhub-2/modules/ROOT/pages/ps-config-fw-rules.adoc

```diff
@@ -82,3 +84,176 @@ image::fw-rule-delete.png[Delete icon on the Firewall rules tab]
 
 . Click *Save Changes* or *Discard changes*.
 . Restart your application.
+
+== Configure TCP Ports for Apps
```

Comment:
It seems we are missing the steps here on how to configure these. Should we have an image too for the UI as guidance?

Reply:
These steps are in Firewall Rules configuration.